### PR TITLE
Retrain fawiki articlequality model

### PR DESCRIPTION
**Pull request by kevinbazira**
Having added sfn templates to fawiki features in T314302, the fawiki articlequality model has been retrained so that it can perform better when ref tags are changed to sfn templates.

Bug: T317531

### CODE REVIEW

**Comment by elukey** [comment]
Kevin everything looks good, I am very ignorant about the changes so I'll let Aiko to comment.

Have you tested the new model on its isvc on Docker? All working fine?
IIUC after this merge we'll have to:
1) Build a new version of this package in Pypi (see articlequality/about.py).
2) Make sure that the repo gets mirrored to its gerrit mirror, and probably bump the submodule sha in the ores deploy repository.
3) Upload the model to Swift etc.. and update the isvc in Lift Wing
4) Deploy to ORES

Is it the plan? Or am I missing something?

**Comment by kevinbazira** [comment]
Yes, Luca, I tested the new model in docker and shared the results here: https://phabricator.wikimedia.org/T317531#8279625

After this change has been merged we will have to deploy this model on both ORES and LiftWing.

**Review by elukey — APPROVED** [review]

**Pull request merged**
